Packs/GCP-Enrichment-Remediation/Playbooks/playbook-Cloud_Credentials_Rotation_-_GCP.yml

```diff
@@ -0,0 +1,805 @@
+          root: GCPIAM.ServiceAccount
+          filters:
+          - - operator: isEqualString
+              left:
+                value:
+                  simple: GCPIAM.ServiceAccount.email
+                iscontext: true
+              right:
+                value:
+                  simple: inputs.serviceAccountEmail
+                iscontext: true
+          accessor: name
+          transformers:
+          - operator: FirstArrayElement
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 890,
+          "y": 1060
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "26":
+    id: "26"
+    taskid: f1fec12d-af5e-4f93-8a4b-5285fad4314a
+    type: regular
+    task:
+      id: f1fec12d-af5e-4f93-8a4b-5285fad4314a
+      version: -1
+      name: Generate Password
+      description: "This function generates a password and allows various parameters to customize the properties of the password depending on the use case (e.g. password complexity requirements).  The default behavior is to generate a password of  *random length* including all four character classes (upper, lower, digits, symbols) with at least five and at most ten characters per class. \n\nThe min_* values all default to 0. This means that if the command is executed in this way:\n!GeneratePassword max_lcase=10\nIt is possible that a password of length zero could be generated. It is therefore recommended to always include a min_* parameter that matches. \n\nThe debug parameter will print certain properties of the command into the WarRoom for easy diagnostics."
+      scriptName: GeneratePassword
+      type: regular
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "13"
+    scriptarguments:
+      min_digits:
+        simple: "5"
+      min_lcase:
+        simple: "5"
+      min_symbols:
+        simple: "1"
+      min_ucase:
+        simple: "1"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 2050,
+          "y": 730
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "27":
+    id: "27"
+    taskid: 1d2baeae-53ce-4e72-86d1-b743af53f7f2
+    type: regular
+    task:
+      id: 1d2baeae-53ce-4e72-86d1-b743af53f7f2
+      version: -1
+      name: List service accounts
+      description: 'Lists service accounts in project, or retrieves a specific service accounts information. One of the arguments: ''service_account_name'' or ''project_name''  must be provided.'
+      script: '|||gcp-iam-service-accounts-get'
+      type: regular
+      iscommand: true
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "25"
```

Comment:
may I suggest adding a conditional task that will check if the required service account was returned? cause if the account isn't there the PB might fail later on ( as you are using filters and transformers in the conditions).

Reply:
added